### DISCUSSION TITLE
원시 값의 동등성 비교는 어떻게 이루어지나요?

### DISCUSSION THREAD

**Discussion by publdaze** [publdaze/js-study#63] [원시 값과 객체의 비교]
- 원시 값의 동등성 비교는 어떻게 이루어지나요?
  - == 연산자와 === 연산자의 차이는 무엇인가요?

**Comment by pipisebastian** [comment]
> 원시 값의 동등성 비교는 어떻게 이루어지나요?

- `===` 일치연산자, `==` 동등 연산자

> == 연산자와 === 연산자의 차이는 무엇인가요?

- `===` 원시값의 타입과 값 모두 비교. 타입변환이 일어나지 않아 안전.
- `==` 원시값의 값만 비교 → 암묵적 타입변환이 이뤄진다.

**Comment by publdaze** [comment]
> - 원시 값의 동등성 비교는 어떻게 이루어지나요?
- 원시 값은 동등 연산자와 일치 연산자를 이용해 비교 가능
> - == 연산자와 === 연산자의 차이는 무엇인가요?
- 동등 연산자는 암묵적 타입 변환이 일어나며 변환된 값들이 일치하는 지를 판별하고
- 일치 연산자는 타입과 값 모두 일치한지 판별
